sysutils/dmidecode/+POST_INSTALL.post

```diff
@@ -0,0 +1,3 @@
+if /usr/local/etc/rc.d/configd status > /dev/null; then
```

Comment:
automatic of actions.d directory is present, please remove

Reply:
removed